src/pages/ReimbursementAccount/RequestorStep.js

```diff
@@ -89,10 +89,6 @@ class RequestorStep extends React.Component {
      */
     clearErrorAndSetValue(inputKey, value) {
         const renamedFields = {
-            street: 'requestorAddressStreet',
```

Comment:
I deleted this because it looks like our old renaming and I see no reason to keep these entries.